soccer/WindowEvaluator.cpp

```diff
@@ -227,7 +235,7 @@ void WindowEvaluator::fill_shot_success(Window& window, Point origin) {
     auto shot_vector = window.segment.center() - origin;
     auto shot_distance = shot_vector.mag();
 
-    // get the angle between the shot vector and the target segment, then
+    // get the angle between the shot std::vector and the target segment, then
```

Comment:
lol, this should actually be vector

Reply:
oh woops my find and replaced didn't work out